pom.xml

```diff
@@ -70,6 +79,11 @@
             <version>2.21.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.8</version>
+        </dependency>
```

Comment:
Could you replace Lombok in favor of Immutables (https://immutables.github.io)? 
I know Lombok is kind of a standard now but personally I don't like it (have reasons, but probably this is not a place for such discussions).